mailgun/views.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import json
+
+from django.http import HttpResponse
+from django.views.decorators.http import require_http_methods
+
+
+logger = logging.getLogger(__name__)
+
+
+@require_http_methods(['GET'])
+def auth_error(request):
+    return HttpResponse(
```

Comment:
Any reason not to use `[JsonResponse](https://docs.djangoproject.com/en/1.8/ref/request-response/#jsonresponse-objects)`?